src/rules/no-standalone-expect.ts

```diff
@@ -0,0 +1,122 @@
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+import {
+  TestCaseName,
+  createRule,
+  isDescribe,
+  isExpectCall,
+  isFunction,
+  isTestCase,
+} from './tsUtils';
+
+const getBlockType = (
+  stmt: TSESTree.BlockStatement,
+): 'function' | 'describe' | null => {
+  const func = stmt.parent;
+
+  /* istanbul ignore if */
+  if (!func) {
+    throw new Error(
+      `Unexpected BlockStatement. No parent defined. - please file a github issue at https://github.com/jest-community/eslint-plugin-jest`,
+    );
+  }
+  // functionDeclaration: function func() {}
+  if (func.type === AST_NODE_TYPES.FunctionDeclaration) {
+    return 'function';
+  }
+  if (isFunction(func) && func.parent) {
+    const expr = func.parent;
+    // arrowfunction or function expr
+    if (expr.type === AST_NODE_TYPES.VariableDeclarator) {
+      return 'function';
+    }
+    // if it's not a variable, it will be callExpr, we only care about describe
+    if (expr.type === AST_NODE_TYPES.CallExpression && isDescribe(expr)) {
+      return 'describe';
+    }
+  }
+  return null;
+};
+
+const isEach = (node: TSESTree.CallExpression): boolean => {
+  if (
+    node &&
+    node.callee &&
+    node.callee.callee &&
```

Comment:
You need to check the type of the node before you do this, as only `CallExpression` nodes have `callee`

Reply:
Ah, makes sense. I just thought checking if they existed would be enough, will address these two things tomorrow 😄

Reply:
That's actually a "is truthy" check :)

You can check property existence via `in`: `'callee' in node.callee`, which can be useful when there are multiple expressions that could have that property (and that we don't care about the type specifically).

However since in this case `callee` is only found on `CallExpression` nodes, it's best to check for that type specifically.